markdown/usage/local_installation.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Local Configuration of pipelines
+subtitle: How to configure your local system to run nf-core pipelines.
+---
+
+If you are running the respective pipeline in a local environment, we highly recommend using either Docker or Singularity.
+
+## Docker
+Docker is a great way to run nf-core pipelines, as it manages all software installations and allows the pipelines to be run in an identical software environment across a range of systems.
+
+Nextflow has [excellent integration](https://www.nextflow.io/docs/latest/docker.html) with Docker, and beyond installing the two tools, not much else is required. The `docker` profile provides a configuration profile for docker, making it very easy to use. This also comes with the required presets to use the AWS iGenomes resource, meaning that if using common reference genomes you just specify the reference ID and it will be automatically downloaded from AWS S3 to your local system.
+
+First, install docker on your system: [Docker Installation Instructions](https://docs.docker.com/engine/installation/)
+
+Then, simply run the analysis pipeline:
+
+```bash
+nextflow run nf-core/YOUR_PIPELINE_NAME -profile docker --genome '<genome ID>'
```

Comment:
I think we probably need more preamble here, as not all pipelines use iGenomes.

Reply:
Done :-)